bundles/org.openhab.ui/web/src/css/app.styl

```diff
@@ -140,6 +140,15 @@ html
    .delete
      --f7-bars-link-color #ff3b30
 
+// Fix actions-modal/popup is placed behind the notch or outside of the screen on iPhone in openHAB iOS app
+@media (orientation: portrait)
+  .ios .actions-modal
+    max-height: calc(100% - env(safe-area-inset-top, 0px) - env(safe-area-inset-bottom, 0px))
```

Comment:
There are f7 CSS variables (https://v5.framework7.io/docs/css-variables#app-core) for that: instead of
`env(safe-area-inset-*)`
I would prefer
`var(--f7-safe-area-*)`